subsys/zigbee/Kconfig

```diff
@@ -266,12 +260,6 @@ config ZIGBEE_SHELL_CTX_MGR_ENTRIES_NBR
 
 endif #ZIGBEE_SHELL
 
-config RADIO_STATISTICS
```

Comment:
Please, do not forget to make a PR changing the `zb_osif_platgform.h` file to `platform_ncs` in ZOI  :slightly_smiling_face:

Reply:
Bump.